CMakeLists.txt

```diff
@@ -33,21 +32,21 @@ add_library(pointcloud_filters src/pointcloud_filters.cpp)
 target_link_libraries(pointcloud_filters ${catkin_LIBRARIES})
 
 add_library(laser_scan_filters src/laser_scan_filters.cpp src/median_filter.cpp src/array_filter.cpp src/box_filter.cpp)
-target_link_libraries(laser_scan_filters ${catkin_LIBRARIES} ${Boost_LIBRARIES} ${Eigen_LIBRARIES})
+target_link_libraries(laser_scan_filters ${catkin_LIBRARIES})
```

Comment:
The information passed to `catkin_package()` are used to export these to downstream packages. They don't affect the current CMake file. `catkin_LIBRARIES` only contains what has been found with `find_package(catkin ...)` above (https://github.com/ros-perception/laser_filters/pull/39/files#diff-af3b638bc2a3e6c650974192a53c7291R11).